app/ui-react/syndesis/src/modules/data/pages/VirtualizationsPage.tsx

```diff
@@ -79,12 +79,20 @@ export function getVirtualizationsHref(baseUrl: string): string {
 export const VirtualizationsPage: React.FunctionComponent = () => {
   const appContext = React.useContext(AppContext);
   const { t } = useTranslation(['data', 'shared']);
-  const { 
-    handleDeleteVirtualization, 
-    handlePublishVirtualization, 
-    handleUnpublishServiceVdb 
+  const {
+    handleDeleteVirtualization,
+    handlePublishVirtualization,
+    handleUnpublishServiceVdb,
   } = VirtualizationHandlers();
-  const { resource: data, hasData, error } = useVirtualizations();
+  const [disableUpdates, setDisableUpdates] = React.useState(false);
+  const { resource: data, hasData, error } = useVirtualizations(disableUpdates);
+
+  // unmount: turn off polling
```

Comment:
This shouldn’t be done here, but in the cleanup function of the effect that starts the polling in first place

Reply:
I'll fix like we talked about and submit a new PR @riccardo-forina.